layouts/partials/page-meta-lastmod.html

```diff
@@ -0,0 +1,2 @@
+{{ T "post_last_mod"}} {{ .Lastmod.Format .Site.Params.time_format_default }}{{ with .GitInfo }}: <a  href="{{ $.Site.Params.github_repo }}/commit/{{ .Hash }}">{{ .Subject }} ({{ .AbbreviatedHash }})</a>{{end }}
```

Comment:
Not that big a deal, but are very long lines like this in code the norm for `cht-docs`? I see we don't do line breaks on content (I assume folks use softwrap in their IDEs), but I wasn't sure about code/template. 

/me goes to read docs formatting docs...

Reply:
The newly added partial is actually just a version of the [existing partial from the theme](https://github.com/google/docsy/blob/master/layouts/partials/page-meta-lastmod.html), with the page links appended -- so this line is taken verbatim from the theme. We don't have any recommendation for line length in `cht-docs`, likely as you mention because the pages are mostly markdown or html which have lots of paragraph text and don't make sense to explicitly wrap. That said, if a page is heavy with new code I think formatting it with appropriate line breaks still makes sense. This partial could have used line breaks for clarity, but I'd rather leave it the way it was in the original theme file.